tests/tests_unit/test_utils/test_time.py

```diff
@@ -576,3 +577,36 @@ def test_to_pandas_freq(granularity: str, start: str, expected_first_step: str):
         # Assert
         actual_index = pd.date_range(start, periods=2, freq=freq)
         pd.testing.assert_index_equal(actual_index, expected_index)
+
+
+def pandas_date_range_tz_data():
+    try:
+        ZoneInfo = import_zoneinfo()
+    except CogniteImportError:
+        return []
+    oslo = ZoneInfo("Europe/Oslo")
+    yield pytest.param(
+        datetime(1916, 8, 1, tzinfo=oslo),
+        datetime(1916, 12, 1, tzinfo=oslo),
+        "1month",
+        5,
+        id="Trigger AmiguousTimeError",
+    )
+
+
+class TestPandasDateRangeTz:
+    @staticmethod
+    @pytest.mark.dsl
+    @pytest.mark.parametrize(
+        "start, end, granularity, expected_length",
+        list(pandas_date_range_tz_data()),
+    )
```

Comment:
This parametrize has just a single test case. Could just merge the content of `pandas_date_range_tz_data` into the test? Then you also don't need the import guard since the test is marked `dsl`.